.golangci.yml

```diff
@@ -1,4 +1,5 @@
-# run:
+run:
```

Comment:
golangci doesn't work anymore afaik

Reply:
We replacec golangci.com with reviewdog. Contributors are still asked to lint their code with `golangci-lint` and this configuration file is read automatically by `golangci-lint` on startup.